pkg/strategy/supertrend/strategy.go

```diff
@@ -92,14 +200,7 @@ type Strategy struct {
 	bbgo.StrategyController
 
 	// Accumulated profit report
-	accumulatedProfit   fixedpoint.Value
-	accumulatedProfitMA *indicator.SMA
-	// AccumulatedProfitMAWindow Accumulated profit SMA window
-	AccumulatedProfitMAWindow int `json:"accumulatedProfitMAWindow"`
-	dailyAccumulatedProfits   types.Float64Slice
-	lastDayAccumulatedProfit  fixedpoint.Value
-	// AccumulatedProfitLastPeriodWindow Last period window of accumulated profit
-	AccumulatedProfitLastPeriodWindow int `json:"accumulatedProfitLastPeriodWindow"`
+	AccumulatedProfitReport AccumulatedProfitReport `json:"accumulatedProfitReport"`
```

Comment:
use pointer type for this field? and check it in the Run method? e.g. `if x == nil { x = newXXX }`

Reply:
done